index.js

```diff
@@ -69,23 +75,176 @@ module.exports = function (homebridge) {
         .then((json) => {
           if (utils.isMessageOfTypeValues(json)) {
             // bulk update ("request values")
-            this.deviceStates = {};
-            for (let item of json) {
-              for (let device of item.devices) {
-                this.deviceStates[device] = item.values.state === 'on';
-              }
-              this.log(`${this.id}: Updated internal states of devices: ${item.devices}`);
+            var item = json.find((item) => {
+              return item.devices.indexOf(this.config.deviceId) !== -1;
+            });
+            if (item) {
+              this.deviceState = item.values.state === 'on';
+              this.log(`Initialized device with state "${item.values.state}"`);
+            } else {
```

Comment:
Expected an identifier and instead saw '}'.<br>Expected ')' and instead saw 'else'.<br>Missing semicolon.